Userland/Applications/FontEditor/MainWidget.cpp

```diff
@@ -110,11 +111,14 @@ ErrorOr<void> MainWidget::create_actions()
     m_open_action = GUI::CommonActions::make_open_action([&](auto&) {
         if (!request_close())
             return;
-        Optional<DeprecatedString> open_path = GUI::FilePicker::get_open_filepath(window(), {}, "/res/fonts/"sv);
-        if (!open_path.has_value())
+
+        auto result = FileSystemAccessClient::Client::the().open_file(window(), "Open font file...", "/res/fonts"sv);
```

Comment:
No need for ellipses in the title. FilePicker itself is the fulfillment of the action text's ellipsis.